Mathlib/Algebra/Associated.lean

```diff
@@ -728,12 +728,13 @@ theorem Associated.of_pow_associated_of_prime' [CancelCommMonoidWithZero α] {p
   (h.symm.of_pow_associated_of_prime hp₂ hp₁ hk₂).symm
 #align associated.of_pow_associated_of_prime' Associated.of_pow_associated_of_prime'
 
-lemma Prime.coprime_iff_not_dvd' [CancelCommMonoidWithZero α] {p n : α} (hp : Prime p) :
+/-- See also `Irreducible.coprime_iff_not_dvd`. -/
+lemma Irreducible.coprime_iff_not_dvd' [Monoid α] {p n : α} (hp : Irreducible p) :
```

Comment:
I don't think you need to add a prime to the name because there is no clash with an existing name.

Reply:
In fact there is a clash, see [here](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/PrincipalIdealDomain.html#Irreducible.coprime_iff_not_dvd).